babel/src/test/java/org/apache/calcite/test/BabelParserTest.java

```diff
@@ -1233,6 +1233,24 @@ private void checkParseInfixCast(String sqlType) {
         + "RETURN (CURRENT_DATE + 1)";
     sql(sql).ok(expected);
   }
+  @Test public void testTranslateUsingCharacterSet() {
```

Comment:
nit: Add new line above.

Reply:
Fix in https://github.com/googleinterns/calcite/pull/84/commits/0fed7d406fda89140c00de15d2d065407e75809a